src/LiveComponent/assets/dist/live_controller.js

```diff
@@ -2154,6 +2154,7 @@ class RequestBuilder {
         const fetchOptions = {};
         fetchOptions.headers = {
             Accept: 'application/vnd.live-component+html',
+            'X-Requested-With': 'XMLHttpRequest',
```

Comment:
Why the value `XMLHttpRequest'`?

Reply:
Set the request like a AJAX request. It's the only way I found so that it does not store it in the [setTargetPath](https://github.com/symfony/symfony/blob/2bac801f8a4cba02e8ecc840fa7bf512034f860a/src/Symfony/Component/Security/Http/Firewall/ExceptionListener.php#L223)